python-phoenixdb/dev-support/make_rc.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+############################################################################
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+############################################################################
+set -e
+
+echo "Script that assembles all you need to make an RC."
+echo "It generates source tar in release directory"
+echo "Presumes that you can sign a release as described at https://www.apache.org/dev/release-signing.html"
+echo "Starting...";sleep 2s
+
+# Set directory variables
+DIR_ROOT="$(cd $(dirname $0);pwd)/.."
+echo $DIR_ROOT
+cd $DIR_ROOT
+VERSION=$(cat setup.py | grep '^version = ".*"$' | grep -o '".*"' |sed 's/"//g')
+
+echo $VERSION
+DIR_REL_BASE=$DIR_ROOT/release
+echo $DIR_REL_BASE
+DIR_REL_ROOT=$DIR_REL_BASE/python-phoenixdb-$VERSION
+echo $DIR_REL_ROOT
+REL_SRC=python-phoenixdb-$VERSION-src
+echo $REL_SRC
+DIR_REL_SRC_TAR_PATH=$DIR_REL_ROOT/src
+echo $DIR_REL_SRC_TAR_PATH
+
+git clean -fx .
```

Comment:
Warn people that this will overwrite changes in the entire repository first?

Reply:
Good idea. Done.